modules/extensions/src/main/java/org/apache/synapse/mediators/bsf/CommonScriptMessageContext.java

```diff
@@ -0,0 +1,848 @@
+/*
```

Comment:
I have changed the license header. What has happened was that this class is only renamed by me.